### ISSUE TITLE
Oculus Crashes on world startup

### ISSUE THREAD

**Issue by conex1** [jrbudda/Vivecraft_115#6]
Every time I load into a world, the oculus will crash by saying it can't track the movement of my headset. This is the only game that does this, I've tried solutions that partake to this problem, but no other VR games I play does this, and it seems to be a problem with vivecraft itself, but still might be a problem with my headset, I can't figure this out.
It's not my specs, as I have the recommended for Oculus, and run plenty of other games fine. I have even tried going into the public test channel, that didn't work either.

**Comment by qfoxb** [comment]
Happening to me too, i have the rift s and it just totally bugged out and now it's convinced it's not connected and it was glitching out really bad

**Comment by Techjar** [comment]
Yeah we've been trying to understand this problem for like a year now. Still no real clue, Oculus is just exploding for no apparent reason.
